packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1424,6 +1424,36 @@ class Transform extends SingleChildRenderObjectWidget {
         assert(scale == null || (scaleX == null && scaleY == null), "If 'scale' is non-null then 'scaleX' and 'scaleY' must be left null"),
         transform = Matrix4.diagonal3Values(scale ?? scaleX ?? 1.0, scale ?? scaleY ?? 1.0, 1.0);
 
+  /// Creates a widget that flips/mirrors its child along the 2D plane.
```

Comment:
Nit: "Creates a widget that flips/mirrors its child in place along the 2D plane."

I'm thinking that we should probably mention somewhere in these docs that the flip happens "in place", so the widget doesn't get moved.  Or maybe if you don't want to change this first sentence you could mention elsewhere that it defaults to Alignment.center or something like that.  Whatever you think is best.

Reply:
```suggestion
  /// Creates a widget that mirrors its child about the widget's center point.
```

I actually don't think you need to specify that it's a 2D plane: everything in Flutter is assumed to be in 2D. It would be remarkable if it were a 3D transform, which is obviously totally possible with a 4x4 matrix, but all of our other transforms are 2D transforms, so it will be assumed that this one is too.